recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set chost = macos_machine %}
-{% set build_number = 2 %}
+{% set build_number = 102 %}
```

Comment:
This looks good but someone might reset to zero. Can we embed the offset inside the build number in the recipe as a jinja2 expression with `{{ build_number | int + 100 }}`?

Reply:
Good idea. I did it with a `build_offset` variable and a comment that resetting to 0 is not likely to be safe in this feedstock which publishes multiple versions (resetting to 0 will only ever be okay if _all_ output versions are increased to previously unpublished values in the same PR)

Reply:
We could also apply the offset only to the duplicate-version outputs, which I _think_ is `libgfortran` and `libgfortran-devel`, but I can check if that would be preferable.